packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -593,8 +596,10 @@ async function handler(req: CustomRequest) {
     }
 
     // Posible to refactor TODO:
-    const paymentApp = await appStore[paymentAppCredential?.app?.dirName as keyof typeof appStore]();
-    if (!(paymentApp && "lib" in paymentApp && "PaymentService" in paymentApp.lib)) {
+    const paymentApp = (await appStore[
+      paymentAppCredential?.app?.dirName as keyof typeof appStore
+    ]()) as PaymentApp;
+    if (!paymentApp.lib?.PaymentService) {
```

Comment:
```suggestion
    if (!paymentApp?.lib?.PaymentService) {
```
That type error is seriously weird. Not focussing on it, just making the code runtime error free